MapboxNavigation/NavigationViewController.swift

```diff
@@ -614,9 +618,6 @@ extension NavigationViewController: NavigationServiceDelegate {
                           direction: location.course,
                           animated: true,
                           completionHandler: {
-                            // Highlight buildings which were marked as target destination coordinate in waypoint.
-                            mapView.highlightBuildings(at: self.routeOptions.waypoints.compactMap({ $0.targetCoordinate }), in3D: self.waypointStyle == .extrudedBuilding ? true : false)
```

Comment:
I think `zoomInAndHighlightBuilding(for:)` name is not really relevant if we start polling and decrease altitude. On the other hand insets will still have to be used to re-center map correctly. What about replacing current implementation with something like:
```swift
// Update insets to be able to correctly center map view after presenting end of route view.
mapViewController.updateMapViewContentInsets(animated: true, completion: {
    // Update user course view to correctly place it in map view.
    self.mapView?.updateCourseTracking(location: location, animated: false)
})
```
and renaming it to something more appropriate?

Reply:
Please also note that we only zoom-in and recenter in case if end of route view is presented. In case of intermediate waypoint such logic will not be performed. I think we'll have to perform more testing to support both cases.

Reply:
What I meant here by renaming current method and removing zoom-in is that if `defaultAltitude` is low enough zoom-in will not be needed at all. In current case we renamed method but still call it internally. I was imagining more like:

```swift
private func frameDestinationArrival(for location: CLLocation?) {
    if waypointStyle == .annotation { return }
    guard let mapViewController = self.mapViewController else { return }
    guard let location = location else { return }
    
    // Update insets to be able to correctly center map view after presenting end of route view.
    mapViewController.updateMapViewContentInsets(animated: true, completion: {
        // Update user course view to correctly place it in map view.
        self.mapView?.updateCourseTracking(location: location, animated: false)
    })
}
```

But for this `defaultAltitude` has to be lowered (in my case `850` doesn't work and buildings are not highlighted at all). With `600` highlighting works. I guess we have to play around with best value.